File: 20160125-1/20160125-1/ViewController.m
//
//  ViewController.m
//  20160125-1
//
//  Created by ChenXin on 16/1/25.
//  Copyright © 2016年 ChenXin. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

- (IBAction)setOutput:(id)sender {
    [[self UserOutput]setText:[[self UserInput] text]];
}
@end

File: 122/122/ViewController.swift
//
//  ViewController.swift
//  122
//
//  Created by ChenXin on 16/3/22.
//  Copyright © 2016年 ChenXin. All rights reserved.
//

import UIKit

class ViewController: UIViewController {


    @IBOutlet weak var urlInput: UITextField!


    @IBOutlet weak var btnGo: UIButton!
    
    @IBOutlet weak var webpage: UIWebView!
    
    @IBAction func btnGoClicked(sender: AnyObject) {
        webpage.loadRequest(NSURLRequest(URL: NSURL(string : urlInput.text!)!))
        
    }
    
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
        
        
    }
    
    
    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

